### DISCUSSION TITLE
Trouble getting this to compile on my machine.

### DISCUSSION THREAD

**Discussion by Zerophase** [tinygrad/tinygrad#1208] [Q&A] [answered]
I'm working on getting this project to compile on Arch Linux using system libraries. I've managed to get to the point where mypy complains about `Only bool return supported for __bool__` when compile.sh is run. 

If I reduce the mypyc command down to `mypyc --check-untyped-defs --explicit-package-bases --warn-unreachable tinygrad/shape/shapetracker.py tinygrad/shape/symbolic.py` I get past mypyc. However, if I add `tinygrad/helpers.py` back in I get the above error message again. I tried checking the `helpers.py` with the mypy plugin for PyCharm, and it says the file is fine. Anyone have any clue what might be causing issues with `helpers.py`?

**Comment by geohot** [comment, marked as answer]
mypyc is not really supported, you don't need to compile it.
